Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1352,7 +1352,7 @@ For more information, ensure that you have the fix `Pull Request <https://github
 Limitations
 ###########
 
-    * Cilium's eBPF kube-proxy replacement currently cannot be used with :ref:`gsg_encryption`.
+    * Cilium's eBPF kube-proxy replacement currently cannot be used with :ref:`encryption_ipsec`.
```

Comment:
Should there be a small addition like: `(however, in can be used in combination with WireGuard)` with ref to wg gsg? Just in case the reader wonders about it whether it was forgotten or not..